app/konnect/analytics/generate-reports.md

```diff
@@ -103,15 +105,16 @@ select **Delete Report**, then confirm deletion in the dialog.
 
 ## Metrics
 
-Each metric depends on a time frame and a primary entity (report type).
+Metric | Description
+-------|------------
+Request count | Total number of API calls within the selected time frame.
+Requests per minute | Number of API calls per minute within the selected time frame.
+Response latency | The amount of time, in milliseconds, that it takes to process an API request. The time starts when the Kong Gateway receives a request and ends when it forwards the response back to the original caller. Users can select between different percentiles (p99, p95, and p50). For example, a 99th percentile response latency of 10 milliseconds means that every 1 in 100 requests took at least 10 milliseconds from request received until response returned. 
+Request size | The size of the request payload received from the client, in bytes. Users can select between different percentiles (p99, p95, and p50). For example, a 99th percentile request size of 100 bytes means that the payload size for every 1 in 100 requests was at least 100 bytes.
+Response size | The size of the response payload returned to the client, in bytes. Users can select between different percentiles (p99, p95, and p50). For example, a 99th percentile response size of 100 bytes means that the payload size for every 1 in 100 response back to the original caller was at least 100 bytes.
 
-Metric | Report type | Description
--------|------------|------------
-Total traffic | Service, route, application | Total number of API calls within the selected time frame.
-Total traffic by status code | Service, route, application | Number of API calls grouped by status code.
-Total traffic by service | Route, application | Number of API calls filtered by services or service versions and grouped by service.
-Total traffic by route | Service, application | Number of API calls grouped by route.
-Total traffic by application | Service, route | Number of API calls grouped by application.
+{:.note}
+> **Note:** {{site.konnect_saas}} Analytics Analytics is using percentiles to enable users to understand the real performance characteristics of their APIs. They have the advantage of distinguishing users who have a good experience from those that have a bad experience. Percentiles can show you a more accurate picture of what most users experience using your API as opposed to hiding critical experiences in an average.
```

Comment:
```suggestion
> **Note:** {{site.konnect_saas}} Analytics uses percentiles to enable you to understand the real performance characteristics of your APIs. Percentiles depict a more accurate picture of what most end users experience using your API instead of hiding critical experiences by displaying averages.
```